src/jitlog/reader.lua

```diff
@@ -53,32 +53,90 @@ typedef struct protobc {
   int length;
   int32_t bc[?];
 } protobc;
+
+typedef struct array_u64 {
+  int length;
+  uint64_t array[?];
```

Comment:
What does `?` mean?

Reply:
It means its dynamic you pass the size of the VLA when you construct the object with ffi.new

Reply:
OK.